smoketest/harness/harness.go

```diff
@@ -236,6 +236,7 @@ func (h *Harness) Start() {
 	h.t.Helper()
 
 	var cfg config.Config
+	cfg.ApplicationName()
```

Comment:
This is a `noop` ApplicationName shouldn't update the struct, and it's not a pointer receiver anyway so the change would be thrown out.
```suggestion
```